### PR TITLE
[SKIP SOF-TEST] Github: update issue template

### PR DESCRIPTION
**Pull request by wszypelt**
Update bug_report with additional information that includes a request to
attach IPC logs with complete IPC Payloads.

Signed-off-by: Wojciech Szypelt <wojciechx.szypelt@intel.com>

### CODE REVIEW

**Comment by wszypelt** [comment]
SOFCI TEST

**Review by marc-hb — CHANGES_REQUESTED** [review]
> SOFCI TEST

Please don't spend hardware cycles testing irrelevant `.github/` changes. I added the `[SKIP SOF-TEST]` prefix to avoid this.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -39,4 +39,5 @@ If applicable, add a screenshot (drag-and-drop an image), or console logs
 (cut-and-paste text and put a code fence (\`\`\`) before and after, to help
 explain the issue.
 
-Please also include the relevant sections from the firmware log and kernel log in the report (and attach the full logs for complete reference). Kernel log is taken from *dmesg* and firmware log from *sof-logger*. See https://thesofproject.github.io/latest/developer_guides/debugability/logger/index.html
+**Logs**
+Please include the relevant sections from the firmware log and kernel log(with complete IPC Payloads) in the report (and attach the full logs for complete reference also IPC payloads are required). Kernel log is taken from *dmesg* and firmware log from *sof-logger*. See https://thesofproject.github.io/latest/developer_guides/debugability/logger/index.html
```
[the remark this answers is not in the document]
Comment (marc-hb) [reply .github/ISSUE_TEMPLATE/bug_report.md:43]: > where and how do you get "IPC payloads"? I don't remember seeing any documentation about this. 

BTW it's a strange to request this in an open-source project when there's no open-source tool or test that can make use of it.

[the remark this answers is not in the document]
Comment (cujomalainey) [reply .github/ISSUE_TEMPLATE/bug_report.md:43]: Ditto on that, IPC payloads is a very high bar, beyond kernel configs. Unless the snooping tools get better this is not acceptable as a requirement

**Comment by plbossart** [comment]
@wszypelt kind reminder to push branches in your own cloned tree, not on the SOF tree directly. Thank you.

**Comment by wszypelt** [comment]
@tmleman @marc-hb @plbossart @cujomalainey  Thanks for taking your time and commenting on this PR. I'll take everything into account. For now, I will close this PR, and perhaps open it in a new correct version after discussions with the team.

**Pull request closed without merging**